src/WebDocs/wwwroot/app/functions/Async/samples/001/content.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="/drapo.js"></script>
+    <title>Function - Async</title>
+</head>
+<body>
+    <br>
+    <span>Async</span>
+    <div d-dataKey="data" d-dataUrlGet="/Sample/GetFunctionPostDataItem" d-dataUrlSet="/Sample/SetFunctionPostDataItemSleep?Sleep=1"></div>
```

Comment:
this 2 APIs are missing

Reply:
I added the two missing APIs but I didn't change the path to CreateNodes, I only added the calls in the Controller and kept the old path.